src/main/java/com/baidu/hugegraph/backend/store/tikv/TikvTable.java

```diff
@@ -168,7 +169,7 @@ protected BackendColumnIterator queryBy(Session session, Query query) {
             assert !query.ids().isEmpty();
             // NOTE: this will lead to lazy create iterator
             return new BackendColumnIteratorWrapper(new FlatMapperIterator<>(
-                    query.ids().iterator(), id -> this.queryById(session, id)
+                    query.ids().iterator(), id -> this.getById(session, id)
```

Comment:
not sure whether the core-test can pass after this switch

Reply:
ok, I have restored